pkgs/by-name/fe/feather/package.nix

```diff
@@ -45,6 +46,7 @@ stdenv.mkDerivation (finalAttrs: {
     libusb1
     openssl
     protobuf
+    python3
```

Comment:
From what I can see `python3` is executed during the build process and it is not needed at runtime. It should go into `nativeBuildInputs`.